safe.html

```diff
@@ -0,0 +1,155 @@
+    <nav class="nav">
+      <div class="logdiv">
+        <img src="./images/logo.jpg" alt="logo" />
+        <a href="">safenet</a>
+      </div>
+      <ul class="nav-ul">
+        <li class="nav-li-active link"><a href="./about.html">About</a></li>
+        <li class="nav-li link"><a href="#">Program</a></li>
+        <li class="nav-li link"><a href="#">join</a></li>
+        <li class="nav-li link"><a href="#"> sponser</a></li>
+        <li class="nav-li link"><a href="#">News</a></li>
+      </ul>
+    </nav>
+    <!-- banner section -->
+    <section class="main">
+      <div id="humb">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+      <div class="content">
+        <h3>"hello there! ever being cyber attacked"</h3>
+        <h1>WE PROTECT YOUR DATA AND NETWORKS FROM ATTACKS</h1>
+        <p>
+          Our Cybersecurity team protect computer networks from cyberattacks and
+          unauthorized access. They do this by trying to anticipate and defend
+          against cyber threats.
+        </p>
+        <h5>SAFENET where all data is safe</h5>
+      </div>
+    </section>
+    <!-- cards section for what we do  -->
+    <section class="tasks">
+      <div class="task-heading">
+        <h4>cyber Attacks</h4>
+        <br>
+      </div>
+      <div class="container">
+        <div class="card">
+          <h3 class="card-heading">
+            <i class="fa-solid fa-database"></i>database
+          </h3>
+          <p class="paragraph">
+            we protect your database from being hacked  and provide support if something happens
+          </p>
+        </div>
+        <div class="card">
+          <h3 class="card-heading">
+            <i class="fa-solid fa-server"></i>servers
+          </h3>
+          <p class="paragraph">
+           we do servers security our team makes sure that all your servers are well protected
+          </p>
+        </div>
+        <div class="card">
+          <h3 class="card-heading">
+            <i class="fa-solid fa-network-wired"></i>networks
+          </h3>
+          <p class="paragraph">
+           we also do network pentesting and ensure safe networks
+          </p>
+        </div>
+        <div class="card">
+          <h3 class="card-heading">
+            <i class="fa-solid fa-cloud-arrow-up"></i>cloud 
+          </h3>
+          <p class="paragraph">
+            we also do cloud security to avoid data breaches in your cloud.
+          </p>
+        </div>
+        <div class="card">
+          <h3 class="card-heading">
+            <i class="fa-solid fa-wifi"></i>wan
+          </h3>
+          <p class="paragraph">
+           we also help companies to protect themselves from wireless or wide area networks attack
+          </p>
+        </div>
+      </div>
+      <div class="last-text">
+        <span>see all services</span>
+        <button type="button">join safenet today</button>
+      </div>
```

Comment:
- [ ] kindly use a brighter and more friendly color here.
![Screenshot (198)_LI](https://user-images.githubusercontent.com/65298048/176853560-4911246e-9199-4c12-8ea6-cfb64047c067.jpg)